src/lib.rs

```diff
@@ -7,7 +7,8 @@
 //! ## Stabalization strategy
 //!
 //! In an effort to release stable 1.0 crates that are forward compatible we are striving
-//! relentlessly to release the bare minimum amount of code.
+//! relentlessly to release the bare minimum amount of code. This 1.0 version currently holds
+//! only two hex parsing functions and the associated error types.
```

Comment:
I'd like this doc to be expanded before release to mention that stable crates should use the error types from this library in their public APIs but can use the 0.x.y version so far for advanced features. If you don't feel like inventing wording I can send a PR later.